custom/gsoc/note/build.gradle

```diff
@@ -0,0 +1,35 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+//
+//plugins {
+//    id 'java'
+//}
+//
+//repositories {
+//    mavenCentral()
+//}
```

Comment:
I am trying to add a custom module for the note API, but it shows the following error message:

```
* What went wrong:
A problem occurred evaluating script.
> Could not find method implementation() for arguments [project ':fineract-core', project ':fineract-provider', org.springframework.boot:spring-boot-starter-web, org.springframework.boot:spring-boot-starter, io.swagger.core.v3:swagger-annotations-jakarta] on object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.
```

I added the commented code and tried to reload, but it shows the following for `starter-web` and other dependencies:

```
:custom:gsoc:note:main: Could not find org.springframework.boot:spring-boot-starter-web:.
Required by:
    project :custom:gsoc:note

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html
```

How can I configure the new module correctly in this situation?

Reply:
I dont think you can define in the dependencies.gradle the dependencies as you did.
Try this one:
```
implementation(project(':fineract-core'))
implementation(project(':fineract-provider'))
implementation('org.springframework.boot:spring-boot-starter')
...
```

Reply:
Same problem
```
* What went wrong:
A problem occurred evaluating script.
> Could not find method implementation() for arguments [project ':fineract-core'] on object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.

```